packages/wdio-applitools-service/src/index.ts

```diff
@@ -72,8 +108,8 @@ export default class ApplitoolsService {
         })
     }
 
-    beforeTest(test) {
-        if (!this.isConfigured) {
+    beforeTest(test?: { title: string, parent: string }) {
```

Comment:
The parameter will always be provided.

Reply:
There is a test that calls this method without any arguments - https://github.com/webdriverio/webdriverio/blob/cc3c86dad3cd2b2636eb45c578ee1e8ac760c100/packages/wdio-applitools-service/tests/service.test.ts#L208

Is it correct?

Reply:
we should adjust the test to pass in a data payload that fits our expectations.